packages/techdocs-common/src/stages/publish/azureBlobStorage.ts

```diff
@@ -289,4 +293,54 @@ export class AzureBlobStoragePublish implements PublisherBase {
       .getBlockBlobClient(`${entityRootDir}/index.html`)
       .exists();
   }
+
+  protected async renameBlob(
+    originalName: string,
+    newName: string,
+    removeOriginal = false,
+  ): Promise<void> {
+    const container = this.storageClient.getContainerClient(this.containerName);
+    const blob = container.getBlobClient(newName);
+    const { url } = container.getBlobClient(originalName);
+    const response = await blob.beginCopyFromURL(url);
+    await response.pollUntilDone();
+    if (removeOriginal) {
+      await container.deleteBlob(originalName);
+    }
+  }
+
+  protected async renameBlobToLowerCase(
+    originalPath: string,
+    removeOriginal: boolean,
+  ) {
+    const newPath = lowerCaseEntityTripletInStoragePath(originalPath);
+    if (originalPath === newPath) return;
+    try {
+      this.logger.debug(`Migrating ${originalPath}`);
+      await this.renameBlob(originalPath, newPath, removeOriginal);
+    } catch (e) {
+      this.logger.warn(`Unable to migrate ${originalPath}: ${e.message}`);
+    }
+  }
+
+  async migrateDocsCase({
+    removeOriginal = false,
+    concurrency = 25,
+  }): Promise<void> {
+    const promises = [];
+    const limiter = limiterFactory(concurrency);
```

Comment:
just curious to why different ways of creating a limiter in here and awsS3?

Reply:
Both are the same default export, but use a different name when importing:
https://github.com/backstage/backstage/blob/34efce9c44ed2fd8414da77089e04f055405d361/packages/techdocs-common/src/stages/publish/awsS3.ts#L24
https://github.com/backstage/backstage/blob/34efce9c44ed2fd8414da77089e04f055405d361/packages/techdocs-common/src/stages/publish/azureBlobStorage.ts#L25